### PR TITLE
Put back nodeport proxy

### PR DESCRIPTION
**Pull request by irozzo-1A**
**What this PR does / why we need it**:
This PR is to include nodeport proxy in the release helm charts tarball. It was previously excluded as the nodeport proxy gets now installed by the operator, but it could still be useful for older Kubermatic installations.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Documentation**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: hack/ci/ci-github-release.sh [hunk 1]
```diff
@@ -68,21 +68,21 @@ function upload_archive {
...
-      err "Response did not contain valid JSON: ${res}"
+    # it the response contain errors
+    if echo "${res}" | jq -e '.errors[0]'; then
+      for err in $(echo "${res}" | jq -r '.errors[0].code'); do
+        # if the error code is 'already_exists' do not fail to make this call
+        # idempotent. To make it better we should alse check that the content
+        # match.
+        [[ "${err}" == "already_exists" ]] && return 0
+      done
+       err "Response contains unexpected errors: ${res}"
```
Comment (xrstf) [inline hack/ci/ci-github-release.sh]: This line is still indented by one too many spaces ;-)

Comment (irozzo-1A) [reply hack/ci/ci-github-release.sh]: indeed

**Pull request merged**